submission/distributor.go

```diff
@@ -34,16 +34,18 @@ import (
 )
 
 var (
-	// Metrics are per-log and add per-endpoint.
+	// Metrics are per-log, per-endpoint and some per-response-status code.
 	once        sync.Once
-	reqsCounter monitoring.Counter   // logid, ep => value
-	rspLatency  monitoring.Histogram // logid, ep => value
+	reqsCounter monitoring.Counter   // logURL, ep => value
+	rspsCounter monitoring.Counter   // logURL, ep, sc => value
```

Comment:
nit: make the comment exactly match the labels in use

Reply:
Done